content/account/users.textile

```diff
@@ -0,0 +1,72 @@
+---
+title: User management
+meta_description: "Learn how to manage users, user roles, and the permissions associated with each role."
+meta_keywords: "users, user management, admin, owner, permission"
+---
+
+The user that creates an Ably account is assigned the account owner role. An account owner has permission to undertake any action within an account, such as inviting new users. There are two other account roles that inherit a subset of an account owner's permissions. An account may only have a single account owner.
+
```

Comment:
```suggestion
The user that creates an Ably account is assigned the account owner role. An account owner has permission to undertake any action within an account, such as inviting new users. There are two other account roles that inherit a subset of an account owner's permissions. An account can only have a single account owner.

```

should we make it clear that it's not a `may` as it isn't possible to have multple account owners? or maybe it means the same?

Reply:
Good point! Fixed.